packages/integration-tests/local.js

```diff
@@ -101,6 +101,7 @@ async function runStep(lambdaPath, lambdaHandler, message, stepName) {
   process.env.CUMULUS_MESSAGE_ADAPTER_DIR = dest;
 
   // add step name to the message
+  // eslint-disable-next-line no-param-reassign
   message.cumulus_meta.task = stepName;
```

Comment:
I know you didn't write this code so I'll approve, but I think we should technically be cloning message into another object before we update it. I think it would be an easy change if you feel inclined.

Reply:
I didn't look at it carefully enough to see it was being passed out of this function. I took care of it.  b12bf2a